### ISSUE TITLE
[accessibility]: add skip links 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1766]
Currently, the site is missing skip links which means that the users who can't use a mouse due to some disability or by choice via a keyboard or switch will have to tab through all the nav links each and every time they visit a page on our site which is not good from an accessibility point of view. 

Sites usually have skip links in place to avoid the unnecessary round trips of the navigation. The below screenshot shows examples of How GitHub and NYT do it.

<img src="https://user-images.githubusercontent.com/46004116/157909311-1c771b6c-89f7-44f0-b25e-35c13307f030.png" width="400" alt="image">

<img src="https://user-images.githubusercontent.com/46004116/157909726-e09a9364-4c67-4492-afca-d69acbfd42be.png" width="400" alt="image">

http://web-accessibility.carnegiemuseums.org/code/skip-link/

**Comment by ChristinFrohne** [comment]
I added a design to the [component library ](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=512%3A377). You can place it similar to how it's placed on Vercel
<img width="90" alt="Bildschirmfoto 2022-03-14 um 09 42 46" src="https://user-images.githubusercontent.com/46345125/158135970-46918f09-1d5a-477e-bbf3-57d688784232.png">

**Comment by nisarhassan12** [comment]
Thanks, Christin sounds good.

**Comment by nisarhassan12** [comment]
@ChristinFrohne I have made Pr https://github.com/gitpod-io/website/pull/1771 for this please review when you get time. Thanks
